gitenberg/dialog.py

```diff
@@ -27,16 +35,35 @@ def ask_generic(self, key, prompt, pwd = False):
                 prompt = '{} >'.format(prompt)
             answer = input( prompt )
         self.answers[key] = answer if answer else self.current.get(key,None)
-    
+
     def ask_username(self):
         self.ask_generic('gh_user', "What is your GitHub username?")
 
     def ask_email(self):
         self.ask_generic('gh_email',"What is your GitHub email?")
 
     def ask_rdf_library(self):
-        self.ask_generic('rdf_library',"What is the path to your PG RDF library?")
 
+        # Ensure we get a y/n answer
+        shouldDownload = None
+        while shouldDownload == None:
+            answer = input("Do you have a local PG RDF library [Y/n]")
+            if answer == "y" or answer == "Y" or answer == "":
+                shouldDownload = False
+            elif answer == "n" or answer == "N":
+                shouldDownload = True
+            
+        if shouldDownload:
+            self.ask_generic('rdf_library',"What is the path to the directory where you would like to have the PG RDF library? (This is a >= 1.1GB file)")
+            if not os.path.exists(self.answers['rdf_library']):
+                os.makedirs(self.answers['rdf_library'])
+            with util.cd(self.answers['rdf_library']):   
+                bz2_rdf = wget.download(RDF_URL)
```

Comment:
I didn't realize that before. Looking into it now, wget still does look like a simpler answer, requests doesn't seem to have a single function which downloads a file, and forces you to manually make sure too much memory isn't used while downloading. But I could switch to requests if the extra dependency is an issue.

Reply:
Oh, so it's because its a big file, I get it.
wget looks harmless. 

https://stackoverflow.com/questions/16694907/how-to-download-large-file-in-python-with-requests-py